### ISSUE TITLE
Crash in getJSDocTags

### ISSUE THREAD

**Issue by ikonst** [webpro/knip#242]
Knip: 2.24.1
Node.js v18.16.1

### Expected
1. Better error reporting. It was unclear what file tripped it.
2. Not crashing.

### Reproduction

**a.ts**
```ts
import { f } from "./b"
```

**b.ts**
```ts
/**
 * @
 */
export function f() {}
```

### Error and stack trace

```
export const getJSDocTags = (node) => ts.getJSDocTags(node).map(node => node.getText().match(/@\S+/)[0]);
                                                                                                    ^

TypeError: Cannot read properties of null (reading '0')
    at file:///<project>/node_modules/knip/dist/typescript/ast-helpers.js:101:101
    at Array.map (<anonymous>)
    at getJSDocTags (file:///<project>/node_modules/knip/dist/typescript/ast-helpers.js:101:61)
    at addExport (file:///<project>/node_modules/knip/dist/typescript/getImportsAndExports.js:100:27)
    at Array.forEach (<anonymous>)
    at visit (file:///<project>/node_modules/knip/dist/typescript/getImportsAndExports.js:136:38)
    at visitNodes (/<project>/node_modules/knip/node_modules/typescript/lib/typescript.js:27934:24)
    at forEachChildInSourceFile (/<project>/node_modules/knip/node_modules/typescript/lib/typescript.js:28556:18)
    at Object.forEachChild (/<project>/node_modules/knip/node_modules/typescript/lib/typescript.js:28019:37)
    at visit (file:///<project>/node_modules/knip/dist/typescript/getImportsAndExports.js:152:12)
    at getImportsAndExports (file:///<project>/node_modules/knip/dist/typescript/getImportsAndExports.js:154:5)
    at ProjectPrincipal.analyzeSourceFile (file:///<project>/node_modules/knip/dist/ProjectPrincipal.js:114:47)
    at analyzeSourceFile (file:///<project>/node_modules/knip/dist/index.js:185:65)
    at file:///<project>/node_modules/knip/dist/index.js:242:17
    at Array.forEach (<anonymous>)
    at main (file:///<project>/node_modules/knip/dist/index.js:241:19)
    at async run (file:///<project>/node_modules/knip/dist/cli.js:24:73)
    at async file:///<project>/node_modules/knip/dist/cli.js:72:1
```

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.25.0. See [Release 2.25.0](https://github.com/webpro/knip/releases/tag/2.25.0) for release notes._

**Comment by webpro** [comment]
Thanks for the report, @ikonst.  Agreed that the error reporting could be improved, but for now I only fixed this specific issue.
